src/localsocketcontroller.cpp

```diff
@@ -343,9 +351,17 @@ void LocalSocketController::parseCommand(const QByteArray& command) {
   logger.warning() << "Invalid command received:" << command;
 }
 
-void LocalSocketController::write(const QJsonObject& json) {
+void LocalSocketController::write(const QJsonObject& json,
+                                  const QString& expect) {
```

Comment:
The `expect` parameter could use a comment. Also, should it and m_messageExpected be renamed to something like expectedResponse?

Reply:
Done!